hole/play.go

```diff
@@ -315,6 +316,15 @@ func play(
 		// is not nil. This seems to be a quirk of the Babashka interpreter
 		// that only occurs when providing code via a command line argument.
 		code += "(print)"
+	case "jq":
+		if hole.ID == "quine" {
+			// Disable implicit output by rejecting input that can be parsed as JSON.
+			var js json.RawMessage
+			if err := json.Unmarshal([]byte(code), &js); err == nil {
+				run.Stderr = "The attempt was worthy."
+				return nil
+			}
+		}
```

Comment:
@sisyphus-ppcg
Apparently, correct. It reminds me of Clojure.
Perhaps something like this?
```go
case "jq":
		if hole.ID == "quine" {
			// Disable implicit output by rejecting input that can be parsed as JSON.
			var js json.RawMessage
			if err := json.Unmarshal([]byte(code), &js); err == nil {
				run.Stderr = "The attempt was worthy."
				return nil
			}
		}
```
Care to give it some tests? It's you that people call the quine wizard, not me. 😅